packages/neuron-ui/src/containers/Navbar/index.tsx

```diff
@@ -84,6 +85,20 @@ const Navbar = () => {
     [networks, networkID, validTarget]
   )
 
+  const [isMigrate, setIsMigrate] = useState(false)
+  useEffect(() => {
+    const dataUpdateSubscription = Migrate.subscribe(migrateStatus => {
+      if (migrateStatus === 'migrating') {
+        setIsMigrate(true)
+      } else {
+        setIsMigrate(false)
+      }
```

Comment:
`setIsMigrate(migrateStatus === 'migrating')` is clear enough, and better to rename `isMigrate` to `isMigrating`